hawkbit-dmf-amqp/src/main/java/org/eclipse/hawkbit/amqp/AmqpSenderService.java

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright (c) 2015 Bosch Software Innovations GmbH and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.hawkbit.amqp;
+
+import java.net.URI;
+
+import org.springframework.amqp.core.Message;
+
+/**
+ * Interface to send a amqp message.
+ */
+@FunctionalInterface
+public interface AmqpSenderService {
+
+    /**
+     * Send the given message to the given uri. The uri contains the (virtual)
+     * host and exchange.
+     * 
+     * @param message
+     *            the amqp message
+     * @param uri
+     *            the reply to uri
+     */
+    void sendMessage(Message message, URI uri);
+
+    /**
+     * Extract the exchange from the uri. Default implementation removes the
```

Comment:
Add an exmaple URI pattern to the javadoc would be easier to know which kind of URI pattern need to be extracted. e.g amqp://<virtualHost>/<exchange>